src/porepy/numerics/nonlinear/nonlinear_solvers.py

```diff
@@ -55,9 +56,10 @@ def solve(self, model) -> tuple[float, bool, int]:
         Returns:
             A 3-tuple containing:
 
-            float:
+            dict:
 
-                The error estimate.
+                Dictionary containing the residual errors and increment errors for each
```

Comment:
Isn't the return value the current errors only?

Reply:
Previously the method returned only the current error, but I changed it so it instead returns the entire dictionary containing the errors from every iteration. Is it important which option you choose here?

Reply:
Again, this comment should now be irrelevant due to existence of `pp.SolverStatistics`.